src/gerbil/runtime/system.ss

```diff
@@ -6,21 +6,52 @@ package: gerbil/runtime
 namespace: #f
 
 (export #t)
-(import "gambit" "util")
+(import "gambit" "util" "control")
 (include "version.ss")
 
+;; Redundantly define this macro here until the version in the prelude is fully bootstrapped.
+(defrules defmutable () ((_ var value) (begin (def var value) (set! var var) (void))))
```

Comment:
initialize with `#!void` again here, you are computing it twice.

Reply:
No. See above.